src/usr/ipmiext/ipmidcmi.C

```diff
@@ -52,6 +159,27 @@ namespace SENSOR
 
         errlHndl_t l_err = NULL;
 
+        // Power Management support check
+        bool support;
+        l_err = getPowerManagementSupportStatus(support);
+        if (l_err != NULL)
+        {
+            // Since the Power Management support information isn`t received,
+            // ignore this error and still try to read the power limit. If the
+            // Power Management is really unsupported, the DCMI Get Power Limit
+            // command will return error code in l_cc
+            TRACFCOMP(g_trac_ipmi,
+                      "Failed to check Power Management support");
+            support = true;
```

Comment:
l_err needs to be committed or deleted to prevent memory leak.  I would suggest committing it as SEV_INFORMATIONAL, since we are still going to try to read the limit directly below.

Reply:
Ok. Done
line 76,88,105,117: rename to ERRL_SEV_INFORMATIONAL
line 167: move collectTrace to here
line 168: add errlCommit